client/src/components/common/footer/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Logo from '../../../assets/icons/Logo';
+
+import './style.css';
+
+const Footer = () => (
+  <footer className="footer container">
+    <section className="app-info">
+      <Logo />
+      <p className="footer__p1">
```

Comment:
Great BEM style. However, class name is not descriptive and do not reflect the intended styling to add to them.

Class names are used to create some sort of "component" pattern for style. For example:
* A class name `btn` indicates that this is the default style for buttons used in our app.
* `btn--rounded` indicates a round button. It would almost only add the attribute `border-radius` to the normal `btn`, and so on.

For our `footer__p1` and `footer__p2`, they almost have the same style. except for the width. You don't need to change these for now, but keep in mind that you can have `footer__p2` with only the `width` attribute, and add both p1 and p2 to the second paragraph.

**Important note**: If you are using percentage or responsive width in general, it's a good idea to set a `min-width` and a `max-width`.